eml/api/userApi.js

```diff
@@ -296,4 +295,17 @@ export const enterNewPassword = async (obj) => {
 	}
 };
 
-
+// Update student study streak
+export const updateStudyStreak = async (studentId) => {
```

Comment:
Maybe I missed it, but is this endpoint ever called?

Reply:
Yes, it is called here in `CompSwipeScreen`:
![image](https://github.com/user-attachments/assets/b83d765d-cc6d-4bae-9f58-ea044f5a6c33)